docroot/modules/custom/open_video_annotation/open_video_annotation.module

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @file
+ * Plugin for Embedding Open Video Annotation.
+ */
+
+/**
+ * Implements hook_page_attachments().
+ */
+function open_video_annotation_page_attachments(array &$page) {
+  // Adds the OpenVideoAnnotation library to the HTML5 player page.
+  if ($node = \Drupal::routeMatch()->getParameter('node')) {
+    if ($node->getType() == 'article') {
+      $page['#attached']['library'][] = 'open_video_annotation/open_video_annotation.load_player';
```

Comment:
Can we not attach the library under the field formatter rather than doing it at page_alter level? It will get executed on all the pages. Field formatter would execute & attach only on pages where the field is getting rendered.